test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -5384,6 +5384,33 @@ class V2MovementsControllerSpec
           )
         }
 
+        "must return Bad Request when unable to find a IE141 message type " in forAll(
```

Comment:
Is this test necessary? 
We don't test for other messages we don't have.

Reply:
Based on previous PR reviews, thought of adding it here.